cmd/api/legal.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func legalTermsOfService(w http.ResponseWriter, r *http.Request) {
+	const text = `<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Chatterino API - Terms of Service</title>
+</head>
+<body>
+  <h1>Chatterino API - Terms of Service</h1>
+  <p>
+    Using the Chatterino API from the Chatterino Client is perfectly fine.
```

Comment:
```suggestion
    Usage of the Chatterino API from the Chatterino Client is perfectly fine.
```
The original is perfectly fine, I just feel like the word `Usage` is very often used in this sort of thing